packages/Ellipsis/src/Ellipsis.tsx

```diff
@@ -0,0 +1,48 @@
+import * as React from 'react';
+import { useRef } from 'react';
+import classNames from 'classnames';
+
+import './ellipsis.css';
+
+export interface EllipsisProps {
```

Comment:
Pour rendre les `props` de l'élément **html** il faut faire un **extend**.
```suggestion
export interface EllipsisProps extends React.ComponentProps<'div'> {
```